src/api/app/models/token/release.rb

```diff
@@ -5,27 +5,50 @@ def call(options)
     set_triggered_at
     return unless options[:package]
 
+    # uniq timestring for all targets
+    time_now = Time.now.utc
+
     # FIXME: Take repository and arch into account
     package_to_release = options[:package]
-    manual_release_targets = package_to_release.project.release_targets.where(trigger: 'manual')
-    raise NoReleaseTargetFound, "#{package_to_release.project} has no release targets that are triggered manually" unless manual_release_targets.any?
+    if options[:targetproject].present? && options[:targetrepository].present? && options[:filter_source_repository].present?
+      source_repository = Repository.find_by_project_and_name(options[:project].name, options[:filter_source_repository])
+      target_repository = Repository.find_by_project_and_name(options[:targetproject], options[:targetrepository])
+      raise CmdExecutionNoPermission, "no permission to write in project #{target_repository.project.name}" unless User.session!.can_modify?(target_repository.project)
```

Comment:
`CmdExecutionNoPermission` is not available in this class, it throws an `uninitialized constant Token::Release::CmdExecutionNoPermission` error

Reply:
I can fix it

Reply:
Fixed